src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.dao.FruitStorageDaoImpl;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+
+    private static final String COMMA = ",";
+    private static final String HEADER = "fruit,quantity";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private FruitStorageDao storageDao = new FruitStorageDaoImpl();
```

Comment:
better to initialize through constructor like you do in `FruitTransactionServiceImpl`